plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -424,6 +447,176 @@ func EqualPriorityMap(_ *v1.Pod, _ interface{}, nodeInfo *schedulercache.NodeInf
 	}, nil
 }
 
+// pickOneNodeForPreemption chooses one node among the given nodes. It assumes
+// pods in each map entry are ordered by their priority.
+// It picks a node based on the following criteria:
+// 1. A node with minimum highest priority victim is picked.
+// 2. Ties are broken by sum of priorities of all victims.
```

Comment:
It means when there are two or more nodes which are equal based on the previous metric, we order them by the second metric.